### PR TITLE
Add simple python3 compatibility fix

### PR DESCRIPTION
**Pull request by mexicarne**
This resolves the package build with python 3.x

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,11 @@
     install_reqs += ['argparse']
     tests_reqs += ['unittest2']
 
-readme = open('README.rst').read()
+if sys.version_info[0] == 3:
```
Comment (ThiefMaster) [inline setup.py]: Better check for `>2`. Assume that a future Python 4 is compatible unless stated otherwise.

**Comment by tony** [comment]
This is a python fix for py-tmuxp @ https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=201564.

@mexicarne Thank you. Want to update it to use ``>2`` like @ThiefMaster mentioned?

**Comment by mexicarne** [comment]
Yes, I just committed the updated check as was suggested.

**Comment by tony** [comment]
@mexicarne Solid. Let's wait for tests to pass, then we'll update the changelog, bump the version and post on pypi. :smile:

**Comment by mexicarne** [comment]
@tony So, all set. Please add some comment with your maintainer approval to bugzilla ticket after you done updating on pypi, so I can update it in FreeBSD ports. Thanks.

**Comment by ThiefMaster** [comment]
@mexicarne not applicable here anymore since the PR is already merged, but generally it's better to amend and force-push simple improvements on a PR instead of adding a separate commit

**Comment by mexicarne** [comment]
@ThiefMaster will do next time, thanks.

**Pull request merged**